### DISCUSSION TITLE
Home media support? (Photos & Videos)

### DISCUSSION THREAD

**Discussion by pollocluck** [jellyfin/Swiftfin#1371] [Questions] [answered]
I was wondering if this will ever be added to the iOS and especially tvOS app anytime in the future, as it seems to be a very straight forward feature to add, unlike music or ebook playback.

**Comment by LePips** [comment, marked as answer]
Please see #1154.

**Reply by pollocluck** [reply]
Thank you for letting me know of this open issue. Is this feature in the works for the next update?
